### PR TITLE
840363 : Update the changes for Ink, Lock-Unlock, Migration and Text-Markups documents.

### PR DESCRIPTION
**Pull request by AntroJames**
Description :
Update the changes for Ink, Lock-Unlock, Migration and Text-Markups document
1. Migration Document
2. Ink Annotation 
3. Textmarkup Annotation

### CODE REVIEW

File: MAUI/PDF-Viewer/Migration.md [hunk 1]
```diff
@@ -432,9 +432,8 @@ To make migration from [Xamarin SfPdfViewer](https://www.syncfusion.com/xamarin-
 <div>{{'[TextMarkupSelectedEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.TextMarkupSelectedEventArgs.html)'| markdownify }} </div>
 </td>
 <td>{{'[AnnotationEventArgs ](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.AnnotationEventArgs.html)'| markdownify }}</td>
-<td></td>
+<td>Provides data for the AnnotationAdded, AnnotationSelected, AnnotationDeselected, AnnotationEdited, AnnotationRemoved events.</td>
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Migration.md:435]: Please link APIs, as it a kind of API documents.

image

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: MAUI/PDF-Viewer/Migration.md (hunk 1 above)
Comment (AntroJames) [reply MAUI/PDF-Viewer/Migration.md]: Updated the feedback.

**Review by Deepak1799 — APPROVED** [review]
Approved the changes

**Pull request merged**